### PR TITLE
Feature/chartjs

### PR DESCRIPTION
**Pull request by VincentKempers**
#57 add chartjs and create a simple chart 
#46 Apply chart in dashboard

### CODE REVIEW

File: src/views/Dashboard/Dashboard.vue [hunk 1]
```diff
@@ -7,155 +7,187 @@
...
+						</div>
+					</a>
+				</li>
+				<li>
+					<a class="verd2" href="/dashboard/verdieping-2">
+						<div>
+							<h3>Floor 2</h3>
+							<render-chart   :width="400" :height="200">
+							</render-chart>
+						</div>
```
Comment (kyunwang) [inline src/views/Dashboard/Dashboard.vue]: Can all be put ina single component

File: src/views/Dashboard/Dashboard.vue [hunk 2]
```diff
@@ -271,5 +277,31 @@ header {
...
+		position: absolute;
+		right: 0;
+		ul {
+			display: flex;
+			flex-direction: column;
+			// padding: 0 1rem;
+			margin: 0;
+			list-style: none;
+			li {
+				// padding: 0 1em;
```
Comment (kyunwang) [inline src/views/Dashboard/Dashboard.vue]: unnecessary comment

File: src/views/Dashboard/Dashboard.vue [hunk 3]
```diff
@@ -271,5 +277,31 @@ header {
...
+				a {
+					text-decoration: none;
+					color: black;
+					padding: 1rem 0rem;
+					div {
+						padding: 0.5em;
+						background-color: black;
```
Comment (kyunwang) [inline src/views/Dashboard/Dashboard.vue]: #4a4a4a for black
#f4f4f4 for white

File: src/views/Dashboard/Dashboard.vue (hunk 1 above)
Comment (VincentKempers) [reply src/views/Dashboard/Dashboard.vue]: Kevin: we do this later 🆗

**Pull request merged**